### ISSUE TITLE
Attribute Error - Function: lookup_team 

### ISSUE THREAD

**Issue by Laneville** [toddrob99/MLB-StatsAPI#91]
Getting an error when running the lookup_team function. Looks like it's in the init py script on line 1221 where it's calling the latest_season function.

<img width="725" alt="Screen Shot 2022-08-13 at 11 02 36 AM" src="https://user-images.githubusercontent.com/19177503/184499755-159c52fa-cffd-4551-bf93-68522f6df000.png">

**Comment by toddrob99** [comment]
Oops! Will release fix shortly.

**Comment by toddrob99** [comment]
Workaround: include sportIds parameter and set the value to `"1"` (string).

``` python
statsapi.lookup_team("chn", sportIds="1")

[{'id': 112,
  'name': 'Chicago Cubs',
  'teamCode': 'chn',
  'fileCode': 'chc',
  'teamName': 'Cubs',
  'locationName': 'Chicago',
  'shortName': 'Chi Cubs'}]
```

**Comment by toddrob99** [comment]
Fixed in v1.5.1, should be available on pypi now.

Thanks for raising the issue, and sorry for the bonehead missed test case.

**Comment by Laneville** [comment]
that was quick :) no no I appreciate the quick fix! Love what you've created with this project
